### PR TITLE
Updating a style to be applied to the federated site.

### PR DESCRIPTION
**Pull request by goynejennifer**
JLG 12012023

**Updating Style in FeatureItems.scss to get the circles in the federated to work properly**
* * *

**JIRA Ticket**: (link) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
This pull request should fix the issue with the 3 circles that are on the federated preproduction site.

# What's the changes? (:star:)
The FeatureItems.scss line 19 was commented out to get the three circles on the federated site working.

Example:
 Updated FeatureItems.scss, line 19, commented out the height because it was messing up the circles.  

# How should this be tested?
To test this out push the updated .scss style file to the preproduction federated website to test if the change does work on the federated website. https://federated-pprd.dlp.cloud.lib.vt.edu/

A description of what steps someone could take to:
None

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
This was branched off the main dev branch.

# Interested parties
Tag (@ mention) interested parties

(:star:) Required fields

### CODE REVIEW

File: .vs/VSWorkspaceState.json
Comment (asthad1) [inline .vs/VSWorkspaceState.json:1]: @goynejennifer I am not sure if this is required for the project. I see your local path so not sure how this is used?

File: .vs/ProjectSettings.json
[the remark this answers is not in the document]
Comment (goynejennifer) [reply .vs/ProjectSettings.json:1]: Not sure why this is set to Null did I set this to null. I probably should be set to a value.

[the remark this answers is not in the document]
Comment (asthad1) [reply .vs/ProjectSettings.json:1]: It came from your merge request. So I guess its you.

[the remark this answers is not in the document]
Comment (goynejennifer) [reply .vs/ProjectSettings.json:1]: LOL.  As far as I know it's just extra files the Visual Studio creates but maybe I don't need to check in just to keep locally.

**Pull request merged**